src/server/scripts/EasternKingdoms/zone_western_plaguelands.cpp

```diff
@@ -76,35 +84,45 @@ class npc_the_scourge_cauldron : public CreatureScript
+                        if (!me->FindNearestCreature(NPC_CAULDRON_LORD_BILEMW, 30.0f))
+                        {
+                            me->SummonCreature(NPC_CAULDRON_LORD_BILEMW, 1728.6443f, -1174.7982f, 59.05936f, 2.356194496154785156f, TEMPSUMMON_TIMED_OR_DEAD_DESPAWN, 600000);
+                            DoDie();
+                        }
                     break;
                 case 200:                                   //dalson
                     if (player->GetQuestStatus(5219) == QUEST_STATUS_INCOMPLETE ||
-                            player->GetQuestStatus(5231) == QUEST_STATUS_INCOMPLETE)
-                    {
-                        me->SummonCreature(11077, 0.0f, 0.0f, 0.0f, 0.0f, TEMPSUMMON_TIMED_OR_DEAD_DESPAWN, 600000);
-                        DoDie();
-                    }
+                        player->GetQuestStatus(5231) == QUEST_STATUS_INCOMPLETE)
+                        //A creature is summoned without being summoned
+                        if (!me->FindNearestCreature(NPC_CAULDRON_LORD_MALVINIOUS, 30.0f))
+                        {
+                            me->SummonCreature(NPC_CAULDRON_LORD_MALVINIOUS, 1865.0482f, -1569.235f, 58.944912f, 3.176499128341674804f, TEMPSUMMON_TIMED_OR_DEAD_DESPAWN, 600000);
+                            DoDie();
+                        }
                     break;
                 case 201:                                   //gahrron
                     if (player->GetQuestStatus(5225) == QUEST_STATUS_INCOMPLETE ||
-                            player->GetQuestStatus(5235) == QUEST_STATUS_INCOMPLETE)
-                    {
-                        me->SummonCreature(11078, 0.0f, 0.0f, 0.0f, 0.0f, TEMPSUMMON_TIMED_OR_DEAD_DESPAWN, 600000);
-                        DoDie();
-                    }
+                        player->GetQuestStatus(5235) == QUEST_STATUS_INCOMPLETE)
+                        //A creature is summoned without being summoned
+                        if (!me->FindNearestCreature(NPC_CAULDRON_LORD_SOULWRATH, 30.0f))
+                        {
+                            me->SummonCreature(NPC_CAULDRON_LORD_SOULWRATH, 1678.6357f, -2278.093f, 58.927708f, 3.543018341064453125f, TEMPSUMMON_TIMED_OR_DEAD_DESPAWN, 600000);
+                            DoDie();
+                        }
                     break;
                 case 202:                                   //writhing
                     if (player->GetQuestStatus(5222) == QUEST_STATUS_INCOMPLETE ||
-                            player->GetQuestStatus(5233) == QUEST_STATUS_INCOMPLETE)
-                    {
-                        me->SummonCreature(11076, 0.0f, 0.0f, 0.0f, 0.0f, TEMPSUMMON_TIMED_OR_DEAD_DESPAWN, 600000);
-                        DoDie();
-                    }
+                        player->GetQuestStatus(5233) == QUEST_STATUS_INCOMPLETE)
+                        //A creature is summoned without being summoned
```

Comment:
```suggestion
                        //A creature is summoned if not already present
```
this makes the comment make more sense. also apply for all the others, please :)

Reply:
also this would make it so that you CAN summon a creature when the npc happens to be fighting some distance (>30.0f) away. Idk if that is the correct behaviour